package.json

```diff
@@ -77,19 +82,19 @@
     "examples:node:commonjs:test": "cd examples/node-commonjs && npm install && npm test",
     "examples:node:esmodules:test": "cd examples/node-esmodules && npm install && npm test",
     "lint": "npm run eslint:check && npm run prettier:check",
-    "eslint:check": "eslint src/ test/ examples/ *.js",
-    "eslint:fix": "eslint --fix src/ test/ examples/ *.js",
+    "eslint:check": "npx eslint src/ test/ examples/ *.js",
```

Comment:
Both npm and yarn makes sure that `node_modules/.bin` is first in the path, is there another reason to add `npx` here? 🤔

Reply:
Just in case. It happens that the modules are not fully installed and then their global versions can be called.